### DISCUSSION TITLE
Default web browser

### DISCUSSION THREAD

**Discussion by javimatia** [pulsar-edit/.github#164] [Pulsar Support] [answered]
Hello, I am using linux, manjaro cinnamon,  and the most recent version of Pulsar in appimage format. I'm starting with Pulsar.

When I open the menu about pulsar or any other, if I click on any link, for example "how to update" or "release notes" it opens them with a python virtual environment browser. The system has firefox configured by default, and I also have chrome installed.

I don't see in the settings where I can set the default browser for Pulsar.
As I said, I'm using manjaro cinnamon, the default browser is firefox and all other apps open firefox, and I'd like to use firefox with Pulsar as well.

If someone can tell me how to configure it, I would appreciate it.

All the best.

**Comment by confused-Techie** [comment]
IIRC it should just open up in the default browser. That's interesting it doesn't, I'll try to have a look and see where this code is specified, but I believe it's within Electron itself. Have you seen this behavior on any other Electron based apps? (e.g. Discord, Bitwarden, Microsoft Teams, Atom)

**Reply by javimatia** [reply]
I'm also using visual studio code which if I'm not mistaken also works with electron. all good in vscode, it opens the links to the help and documentation with firefox without problems.

If it helps I can try whatever is needed.

I have made a correction in the op, it is not flatpak, I am using appimage

Thank you.

**Reply by javimatia** [reply, marked as answer]
Hi, I've been working on this and I think the problem is the way Pulsar searches for the default browser. I'm not quite sure, but it has to do with myme types and the way xdg-open works, and how cinnamon desktop set default apps.
I have created the file ~/.local/share/applications/mimeapp.list
with this content and now Pulsar opens the links with firefox, as I wanted.

  [Default Applications]
text/html=firefox.desktop;
text/xml=firefox.desktop;
x-scheme-handler/http=firefox.desktop;
x-scheme-handler/https=firefox.desktop;

For my part the issue is resolved. A greeting and thanks for the attention, now to continue delving into Pulsar, it seems to me a great project.

**Reply by confused-Techie** [reply]
@javimatia I really appreciate you providing a solution that works for you! And super glad to here you got a fix in, I'll try to take a look to see what we can do to have this behavior match what's expected, and very good to know it works in VSCode, because you are correct that also uses Electron. So now just to find how we do it differently from then, thanks a ton!
